lib/plugins/backup.rb

```diff
@@ -3,8 +3,8 @@
 require_relative '../moonshot/creds_helper'
 
 module Moonshot::Plugins # rubocop:disable Documentation
-  # Monkeypatching TarWriter for our
-  # needs to improve clarity
+  # Defining new method for the ease of use in this
```

Comment:
This doesn't explain what "this particular use case is" -- what does the original method do and why are you changing it? That documentation will definitely save someone time in the future.